book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,141 @@
+People will not be able to invest in this, if the new behaviours are not adequately rewarded (for example, focused placed on publications written instead of open data/software).
+Short term goals can give people a sense of achievement and encourage them to move forward.  
+
+(er-activism-cultural-change-leading)=
+## Leading change
+
+According to John P Kotter ({cite:ps}`Kotter2012change`) there are eight steps in change:
+1.	Establishing a sense of urgency by identifying potential crises/opportunities.
+2.	Creating the guiding coalition, a team that works together to lead the change.
+3.	Developing a vision to direct the change and strategy to achieve this vision.
+4.	Communicating the change vision using every communication channel possible, including role modelling.
+5.	Empowering action by removing obstacles and encouraging risk taking.
+6.	Generating short-term wins and recognise people who made these wins possible.
+7.	Consolidating change: maintain a sense of urgency, use situations to start a new wave of change, keep celebrating successes and evaluate/adjust if needed. 
+8.	Anchoring new approaches in the culture.
+
+The first four steps in the transformation process help to convince people or an organisation that they need to change. 
+Phases five to seven introduce new practices. 
+The last stage focusses on making these new practices stick. 
+
+**Change is a team effort**
+
+A single individual will not affect cultural change in an organisation, as this requires collective action.
+A team with members in position of power, members with expertise or leadership skills and credibility, will be needed. 
+
+**Vision for change**
+
+The **vision** refers to the future and why people should strive to create that future. 
+A vision clarifies the general direction for change and motivates people to take coordinated action. 
+A vision will acknowledge that sacrifices will be needed to achieve benefits that would not be achievable without change. 
+An effective vision is imaginable (it conveys a picture of the future), desirable (appealing to anyone involved in the change), feasible (has attainable goals), focused (for clear decision making), and flexible (to allow for alternative responses in light of changing conditions).
+
+This vision should be easy to communicate, in under five minutes. 
+The effective communication of the vision depends on the simplicity of the message, whether appropriate metaphors or pictures are used, the communication channels used, how often the message is repeated and how the behaviour is modelled by important people. 
+
+
+
+(er-activism-cultural-change-summary)=
+## Summary
+
+Cultural change is a long-term effort that you can achieve by taking one step at a time, ideally with some like-minded individuals.
+As it is difficult to achieve change without knowing where you're heading, it is good to outline a vision of what the change should look like. 
+To achieve change there will need to be transition. 
```

Comment:
Just a suggestion for alternative drafting:
```suggestion
To achieve change, it is inevitable that a transition will occur.
```

Reply:
Think it maybe reads better in the original!

Reply:
I think I might change this to : To achieve change, transitions are inevitable.